src/Persistence_matrix/include/gudhi/Persistence_matrix/columns/naive_vector_column.h

```diff
@@ -224,7 +224,13 @@ class Naive_vector_column : public Master_matrix::Row_access_option,
 };
 
 template <class Master_matrix>
-inline Naive_vector_column<Master_matrix>::Naive_vector_column(Column_settings* colSettings)
+using STD_naive_vector_column = Naive_vector_column<Master_matrix, std::vector<typename Master_matrix::Matrix_entry*> >;
+template <class Master_matrix>
+using Small_naive_vector_column =
+    Naive_vector_column<Master_matrix, boost::container::small_vector<typename Master_matrix::Matrix_entry*, 8> >;
```

Comment:
It seems strange to hardcode 8.

Reply:
I agree. It would be better to have something general here, but I didn't wanted to add an option for that... And 8 was what worked the best in my benchmarks to compute a barcode. Do you see an easy way to give the possibility to customize this value for the user without having to add an option just for this particular case?